src/Sulu/Bundle/AudienceTargetingBundle/Tests/Unit/EventListener/TargetGroupSubscriberTest.php

```diff
@@ -441,7 +429,7 @@ public function testAddTargetGroupHitScript(
         $this->assertEquals('<body><script></script></body>', $response->getContent());
     }
 
-    public static function provideAddTargetGroupHitScript()
+    public static function provideAddTargetGroupHitScript(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes